### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by matpena** [JaouherK/streamDeck-weatherPlugin#28]
Love the app!
But the setting is set to fetch data every 10 minutes, but it's getting refreshed only when I press the button.

Mac OS v. 5.0.0

**Comment by Aetles** [comment]
Same problem here, only updated when pressed.

**Comment by JaouherK** [comment]
@Aetles  Do you have also a macOS?

**Comment by Aetles** [comment]
Yes I do. But the problem was not persistent, after some days I noticed that it worked again. Can't say I've been troubleshooting it enough to see a pattern (like if it was a reboot of the computer or a restart of the Stream Deck app or just time passing by). 

I've also seen times when I just get "NaN" instead of temperature, but that is probably unrelated.

**Comment by JaouherK** [comment]
NaN happened when we received 0°C. This has already merged and waiting for it to be released soon in addition to some other fixes.

**Comment by JaouherK** [comment]
@matpena do you still face this issue?

**Comment by matpena** [comment]
Will re-install and let you know.
